### ISSUE TITLE
[Issue] - next-steps.md states incorrect manifests directory

### ISSUE THREAD

**Issue by michaelmcneilnet** [Azure/azure-dev#4401]
- [x] Make sure you've installed the latest version using [instructions in the wiki](../wiki/install)

**Output from `azd version`**
`azd version 1.10.1 (commit 31409a33266fb4a5fdbb644bc83988e725d6c7c9)`

**Describe the bug**
The generated `next-steps.md` file states 

`In addition, for each project resource referenced by your app host, a containerApp.tmpl.yaml file will be created in a directory named manifests next the project file. This file contains the infrastructure as code for running the project on Azure Container Apps.`

This isn't the case for me. These `tpml.yaml` files are within the `infra/` folder along with the rest of the generated bicep , after running `azd infra synth`. I am using Aspire with packages at `9.0.0-preview.4.24502.4`

**To Reproduce**
1. Run `azd init`
2. Run `azd infra synth`
3. View the `next-steps.md` file.

**Expected behavior**
The `next-steps.md` file states that the `containerApp.tmpl.yaml` files are within the `infra/` folder.

**Comment by rajeshkamal5050** [comment]
@weikanglim can we fix the generated next-steps doc?

**Comment by michaelmcneilnet** [comment]
Explanation at https://learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?branch=main&tabs=windows#generate-bicep-from-net-aspire-project-model also doesn't match what I am seeing, as per my original post

**Comment by weikanglim** [comment]
We'll get the docs updated to reflect the newly introduced product behavior.

**Comment by rajeshkamal5050** [comment]
@weikanglim can we fix this minor generated doc issue? @kristenwomack

**Comment by weikanglim** [comment]
@vhvb1989 https://learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?branch=main&tabs=windows#generate-bicep-from-net-aspire-project-model should we still document this?

**Comment by vhvb1989** [comment]
> [@vhvb1989](https://github.com/vhvb1989) https://learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?branch=main&tabs=windows#generate-bicep-from-net-aspire-project-model should we still document this?

Yes, the basic Aspire support that azd provides still apply here.
Azd is now providing a limited-functionality warning as a way to move folks to a mode where Aspire defines Azure model. But we can't remove this docs

**Comment by rajeshkamal5050** [comment]
@vhvb1989 so we will leave the document as-is and not fix it. Given that we are warning folks to move modeling in the Aspire layer?

**Comment by spboyer** [comment]
Closing as stale — this issue has had no activity for 180+ days. If this is still relevant, please reopen with reproduction steps on the latest version of azd.
